### DISCUSSION TITLE
Chakra installation docs do not work for current version of remix js

### DISCUSSION THREAD

**Discussion by drewg2009** [chakra-ui/chakra-ui#8010] [Q&A]
I tried following this guide https://chakra-ui.com/getting-started/remix-guide but there are files and examples that are only in version 1 of remix, not 2 (the latest version). 

Will there be an update to this/are there docs for the latest version of remix?

**Comment by GOWxx** [comment]
Looking forward to any updates.

**Comment by ivosabev** [comment]
Any movement on this? I see there is a template for Chakra 3 with Remix, but not for Chakra 2
